test/unit/integrations/yuupay/yuupay_helper_test.rb

```diff
@@ -0,0 +1,114 @@
+require 'test_helper'
+
+class YuupayHelperTest < Test::Unit::TestCase
+  include OffsitePayments::Integrations
+
+  def setup
+    @order = 'order-500'
+    @account = 'zork'
+    @key = 'TO78ghHCfBQ6ZBw2Q2fJ3wRwGkWkUHVs'
+    @amount = 123.45
+    @currency = 'USD'
+    @test = false
+    @country = 'US'
+    @account_name = 'Widgets Inc'
+    @transaction_type = 'sale'
+    @options = {:amount => @amount,
+                :currency => @currency,
+                :test => @test,
+                :credential2 => @key,
+                :country => @country,
+                :account_name => @account_name,
+                :transaction_type => @transaction_type}
+    @helper = Yuupay::Helper.new(@order, @account, @options)
+  end
+
+  def test_forward_url
+    assert_equal @helper.forward_url, @helper.credential_based_url
```

Comment:
I'm not sure how I feel about this test. Is there a reason we expect these two different methods to return identical urls?